docs/ldml/tr35.html

```diff
@@ -1147,7 +1147,9 @@ <h4><a name="Canonical_Unicode_Locale_Identifiers" href="#Canonical_Unicode_Loca
               <li>If there are additional subtags in the <i>replacement</i> value, add them to the result, but
                 only if there is no corresponding subtag already in the
                 tag.</li>
-              <li>Five special deprecated grandfathered codes (such as <em>i-default</em>) are in  type attributes, and are also replaced.</li>
+              <li>Five special deprecated <span class="removed">grandfathered codes (</span>
```

Comment:
This is unnecessary, the whole block that includes this is marked for removal. You should look at the html in a browser to see this.